server/src/main/java/com/google/musicanalysis/types/AnalysisGroup.java

```diff
@@ -3,17 +3,23 @@
 import java.util.ArrayList;
 import java.util.HashMap;
 
-public class AnalysisTrio {
+public class AnalysisGroup {
```

Comment:
This should remain "CommentAnalysis" or even "VideoAnalysis" since this class holds data and results obtained from analyzing a video. AnalysisGroup doesn't tell me much about what this class is for and makes it easy to start piling on fields that shouldn't really be here.